cmd/jujud/agent/caasoperator/manifolds.go

```diff
@@ -162,7 +162,6 @@ func Manifolds(config ManifoldsConfig) dependency.Manifolds {
 		upgraderName: caasupgrader.Manifold(caasupgrader.ManifoldConfig{
 			AgentName:            agentName,
 			APICallerName:        apiCallerName,
-			UpgradeStepsGateName: upgradeStepsGateName,
```

Comment:
We don't currently have any upgrade steps targetted at a k8s operator. We used to have (for iaas), unit agent specific upgrade steps to write changes to the unit agent's directories, but these were removed in juju 2. Since for k8s we would just be starting a new pod, I don't think there's any reasonable upgrade steps we need to wait for here. So this seems a logical change to make.